### PR TITLE
 Only mark manual query as User Query SFINT-2449

### PR DESCRIPTION
**Pull request by jeremierobert-coveo**
Only search event that is defined as manual will be tag as a User Query instead of each search event that as a query_expression.

### CODE REVIEW

File: src/utils/events.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const MANUAL_SEARCH_EVENT_CAUSE = Object.freeze([
+    'omniboxAnalytics',
```
Comment (jeremierobert-coveo) [inline src/utils/events.ts]: Here, I not 100% sure which event cause shall be mark as a User Query. I mean for submit is trivial but others...

**Comment by louis-bompart** [comment]
> See SFINT-2449

Yeah nah, not enough, details.

File: src/utils/events.ts (hunk 1 above)
Comment (jeremierobert-coveo) [reply src/utils/events.ts]: When I say not 100% sure mean, I'm open to discussion 😄 .
I passed each listed search cause in the documentation and those are the ones that came out.

Comment (louis-bompart) [reply src/utils/events.ts]: I have no real opinions on the matter to be perfectly honest, just'd like to have a good confidence in it :)

File: src/components/UserActions/Strings.ts [hunk 2]
```diff
@@ -17,8 +17,8 @@ Translation.register(Language.English, {
     UserActivity_other_event: 'Other Event',
     UserActivity_other_events: 'Other Events',
 
-    UserActivity_search: 'User Query',
-    UserActivity_query: 'Query',
+    UserActivity_search: 'Query',
+    UserActivity_query: 'User Query',
```
Comment (tedre191) [inline src/components/UserActions/Strings.ts]: What's changed exactly ?

File: src/components/UserActions/UserActivity.ts [hunk 3]
```diff
@@ -349,6 +350,10 @@ export class UserActivity extends Component {
 
         return cell;
     }
+
+    private isManualSubmitAction(action: UserAction) {
```
Comment (tedre191) [inline src/components/UserActions/UserActivity.ts]: Add a comment that explain what type of events are "manual"

File: tests/components/UserActions/UserActivity.spec.ts [hunk 4]
```diff
@@ -201,20 +201,33 @@ describe('UserActivity', () => {
...
-                    return env;
-                })
-            );
-
-            return delay(() => {
-                const clickElement = mock.cmp.element.querySelector('.coveo-search');
-
-                expect(clickElement).not.toBeNull();
-                expect(clickElement.querySelector<HTMLElement>('.coveo-title').innerText).toBe('User Query');
+        ['omniboxAnalytics', 'userActionsSubmit', 'omniboxFromLink', 'searchboxAsYouType', 'searchboxSubmit', 'searchFromLink'].map(cause => {
```
Comment (tedre191) [inline tests/components/UserActions/UserActivity.spec.ts]: @louis-bompart is gonna love that, but I don't mind.

Comment (louis-bompart) [reply tests/components/UserActions/UserActivity.spec.ts]: I'm the one that asked for it :D

File: src/components/UserActions/Strings.ts (hunk 2 above)
Comment (jeremierobert-coveo) [reply src/components/UserActions/Strings.ts]: The label tagged in the UserActivity component.

**Pull request merged**